src/services/slack_sync.ts

```diff
@@ -94,6 +105,8 @@ const slackSync = {
       cursor = result.response_metadata?.next_cursor;
     } while (cursor);
 
+    logger.info(`Fetched ${messages.length} messages`);
```

Comment:
I've been trying to lean strongly towards putting all the variables into the `{data}` blob.

```suggestion
    logger.info({messages_count: messages.length}, `Fetched messages`);
```
Same for the other messages below.
Maybe our preference should be different in scripts, though.